pkg/accelerator/accelDeviceProvider.go

```diff
@@ -73,9 +73,12 @@ func (ap *accelDeviceProvider) AddTargetDevices(devices []*ghw.PCIDevice, device
 	return nil
 }
 
-func (ap *accelDeviceProvider) GetFilteredDevices(devices []types.HostDevice, rc *types.ResourceConfig) ([]types.HostDevice, error) {
+func (ap *accelDeviceProvider) GetFilteredDevices(devices []types.HostDevice, rc *types.ResourceConfig, selectorIndex int) ([]types.HostDevice, error) {
```

Comment:
For this function, should there be any refactoring to adjust the cyclomatic complexity? I think the other `GetFilteredDevices` functions have `//nolint:gocyclo` for them (e.g., `netDeviceProvider.GetFilteredDevices`). So, I am in favor of adding the same exclusion to this function.

Reply:
im fine with adding it.